pages/study/timer/index.js

```diff
@@ -4,41 +4,33 @@ import {useEffect, useState} from 'react';
 
 export default function Timer() {
   const [typeInterval, setTypeInterval] = useState("Group's Timer");
```

Comment:
Possible renaming to: Session Timer, always had weird issues poppin' out when using **'s**